frontend/src/components/ingress/Details.tsx

```diff
@@ -36,6 +37,29 @@ export default function IngressDetails() {
     );
   }
 
+  /** this function is used to format the url to be used in the Link component */
+  function isFormatURL(url: any) {
+    if (url === '*') {
+      return '*';
+    }
+    try {
+      /** https://developer.mozilla.org/en-US/docs/Web/API/URL/URL
+       * new URL() constructor creates a new URL object instance only if the string passed is a valid URL
+       * if the string is not a valid URL, then an error is thrown
+       * note: adding the protocol to the url string is required for the URL constructor to work because data.host is only the host name */
```

Comment:
Except for the last line (without the note prefix), I don't think the comment is needed as it's kind of the API doc.

Reply:
Fixed it!